### DISCUSSION TITLE
if the camera's time is not sync to aws cloud, is that ok?

### DISCUSSION THREAD

**Discussion by pppaulpeter** [AxisCommunications/acap-integration-examples-aws#23] [General]
if not, i mean probably the aws authentification will failed, and how to solve this problem?

**Comment by mattias-kindborg-at-work** [comment]
For [images-to-aws-s3](https://github.com/AxisCommunications/acap-integration-examples-aws/tree/main/images-to-aws-s3) I don't think there's a problem since the device is authenticated using an access token, but for [telemetry-to-aws-iot-core](https://github.com/AxisCommunications/acap-integration-examples-aws/tree/main/telemetry-to-aws-iot-core) there will probably be a problem if the clocks aren't synchronized.
